app/views/register/establishers/company/director/whatYouWillNeed.scala.html

```diff
@@ -16,24 +16,33 @@
 
 @import config.FrontendAppConfig
 @import uk.gov.hmrc.play.views.html._
-@import controllers.register.establishers.company.routes._
+@import controllers.register.establishers.company.director.routes._
 
-@(appConfig: FrontendAppConfig, schemeName: Option[String], postCall: Call, srn:Option[String], companyName:String)(implicit request: Request[_], messages: Messages)
+@(
+        appConfig: FrontendAppConfig,
+        schemeName: Option[String],
+        srn: Option[String],
+        companyName: String,
```

Comment:
Would it be cleaner to pass the postUrl to the view instead of 3 parameters - mode, estIndex & dirIndex?

Reply:
Done